src/offline/cable_surface_types.F90

```diff
@@ -0,0 +1,36 @@
+!******************************************************************************
```

Comment:
This is more of a general architectural comment for the code - not for you to fix here; but this duplicated code is argument for a "common" directory that contains code (such as this) that are common across the various instance of CABLE (be it coupled/ESMX.X or offline etc.)

This is all extra info for a case I'm building to simplify the structure.

Reply:
As discussed it will be eventually, but for now having application specific versions of cable_surface type means we dont have to spagettify the core science code. Which is two steps away  from where we want to be eventually.